src/test/java/uk/gov/dhsc/htbhf/steps/AddressLookupSteps.java

```diff
@@ -0,0 +1,132 @@
+package uk.gov.dhsc.htbhf.steps;
+
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.openqa.selenium.WebElement;
+import uk.gov.dhsc.htbhf.page.PostcodePage;
+import uk.gov.dhsc.htbhf.page.SelectAddressPage;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.dhsc.htbhf.steps.Constants.POSTCODE;
+import static uk.gov.dhsc.htbhf.steps.Constants.POSTCODE_WITH_NO_RESULTS;
+import static uk.gov.dhsc.htbhf.utils.WiremockResponseTestDataFactory.aPostcodeLookupResponseWithResults;
+
+/**
+ * Steps for looking up addresses for a claimant
+ */
+public class AddressLookupSteps extends CommonSteps {
+
+    @Given("OS places returns an error response")
+    public void givenOsPlacesReturnsAnError() {
```

Comment:
This first method shouldn't be in here, it'll be the first method updated in the next PR, I'll take it out if anyone is bothered or I need to make an update.